doc/user/content/third-party/debezium.md

```diff
@@ -1,36 +0,0 @@
----
```

Comment:
I think we could have kept this page, just rearranged the content around a bit.

Reply:
We could restore it, but it doesn't have any information that isn't in the guide, I don't think.